.drone.yml

```diff
@@ -5187,8 +5187,11 @@ steps:
   image: tezedge/tezos-node-bootstrap:latest
   pull: if-not-exists
   user: root
+  volumes:
+    - name: perf
+      path: /perf
   commands:
-    - tezos-node-bootstrap performance-test --level=$${WRK_TEST_BLOCK_HEADER} --tezedge-old-node=$${TEZEDGE_NODE_TARGET_BRANCH_RPC_CONTEXT_ROOT} --tezedge-new-node=$${TEZEDGE_NODE_RPC_CONTEXT_ROOT} --ocaml-node=$${OCTEZ_NODE_RPC_CONTEXT_ROOT_1} --wrk-test-duration=$${WRK_TEST_DURATION} --max-latency-threshold=100 --throughput-threshold=10
+    - tezos-node-bootstrap performance-test --tezedge-old-node=$${TEZEDGE_NODE_TARGET_BRANCH_RPC_CONTEXT_ROOT} --tezedge-new-node=$${TEZEDGE_NODE_RPC_CONTEXT_ROOT} --ocaml-node=$${OCTEZ_NODE_RPC_CONTEXT_ROOT_1} --wrk-test-duration=$${WRK_TEST_DURATION} --max-latency-threshold=100 --throughput-threshold=10 --latency-no-fail --throughput-no-fail --url-file=/perf/wrk-rpc-list-delphi.txt
```

Comment:
@akoptelov  @tizoc 
is this change ok?

Reply:
Yes, this makes the new `tezedge/tezos-node-bootstrap` work for this step. I forgot to update this line, now this is in synch with this one:
https://github.com/tezedge/tezedge/blob/26aeaa32cb2fdf53187d1e278b541799e233ee83/.drone.yml#L5170

Reply:
@akoptelov @bkontur other than what Alexander mentioned, there is one thing I changed here that may be controversial, I added `--latency-no-fail --throughput-no-fail` so that the test finishes to completion even when there is a latency or throughput regression (without those, the test will fail at the first regression and not continue).

Reply:
@tizoc @akoptelov 
what about this? do we need to do something more?
if we have any regression, will this tests fail somehow or not?

Reply:
@bkontur something better is needed. Latencies are not sable enough for this to not randomly fail when somebody starts a branch that changes something completely unrelated. Also, we cannot improve everything all the time, sometimes tradeoffs are needed (and sometimes numbers may be good because they are producing the wrong answer fast, and fixing that will cause a regression).

I just don't like the current model that considers everything a failure, I think it is the wrong approach. What we need is some good visualization, and it will be up to an human to decide if the changes are acceptable or not.

Reply:
@tizoc @akoptelov 
ok, could you please create a jira under to tech.debt for that latencies, latencies are hot issue

Reply:
Missed this yesterday. Ok, will do.